package/yast2-drbd.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Nov 21 08:30:41 UTC 2017 - nwang@suse.com
+
+- bsc#1069131, compat udev-always-with-vnr and quorum.
+- bsc#1069132, fix error message when write resource conf to disk.
+- Version 3.3.0
```

Comment:
Just a note; we decided to use 4.0.0 for SLE15 (master), just change it here and in the spec file.

Reply:
Got it.

Reply:
Just for the record: see http://yastgithubio.readthedocs.io/en/latest/versioning/ for the details